build.gradle

```diff
@@ -25,9 +30,10 @@ def todaysDate = LocalDate.now().format('yyyy-MM-dd')
 version = System.getenv('CI') ? "${todaysDate}.${System.getenv('CIRCLE_BUILD_NUM')}" : todaysDate
 
 jar {
-    baseName = rootProject.name
```

Comment:
I'd noticed that `baseName` was deprecated, didn't really know what to do about it :+1: